catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -475,11 +474,8 @@ public static void invalidateLoginTokenIfUserRestricted(Context context) {
 
 	public static void logoutUser(Context context) {
 		SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
-		String userName = sharedPreferences.getString(Constants.USERNAME, Constants.NO_USERNAME);
 		GoogleLoginHandler googleLoginHandler = new GoogleLoginHandler((AppCompatActivity) context);
 		googleLoginHandler.getGoogleSignInClient().signOut();
-		LogoutTask logoutTask = new LogoutTask(context, userName);
```

Comment:
WebViewActivity.clearCookies(); already does everything that is necessary to be logged out in the webview.